### ISSUE TITLE
Website Accessibility

### ISSUE THREAD

**Issue by not-my-profile** [jdavis/twofactorauth#6215]
Hey, nice project :) Just FYI you could improve the accessibility of your website with the following simple changes.

**keyboard accessibility** (website should be navigable with just `Tab`/`Shift+Tab` and `Enter`)

* add `tabindex=0` to `regionDropdown`
* turn the category buttons into actual links e.g. `href=#backup` and expand the tables by listening to `window.onhashchange` instead of the `click` event

**screen reader accessibility**

* add `aria-label` to the icons in the table

The social buttons are also inaccessible. You could add `tabindex=0 role=button` but even better would be turning them into regular links.

I unfortunately don't have time to implement these changes; only for reporting^^ But if you have any questions I can try to answer them :)

**Comment by Carlgo11** [comment]
Seems the requested changes have all been added so I'll close this issue
